rotkehlchen/tests/api/test_pnl_csv.py

```diff
@@ -177,7 +162,71 @@ def test_history_export_download_csv(
         tempzipfile.write_bytes(response.content)
         with zipfile.ZipFile(tempzipfile, 'r') as zip_ref:
             zip_ref.extractall(extractdir)
-        assert_csv_export_response(response, extractdir, is_download=True, unicode_check=True)
+        assert_csv_export_response(response, extractdir, is_download=True)
+
+
+@pytest.mark.parametrize('mocked_price_queries', [{'ETH': {'EUR': {1569924574: 1}}}])
+@pytest.mark.parametrize('encoding_to_use', ['utf-8', 'cp1252'])
+def test_encoding(
+        rotkehlchen_api_server,
+        tmpdir_factory,
+        encoding_to_use,
+):
+    """Test that exporting csv and debug report works correctly with different encodings"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    locale_patch = patch.object(
+        _bootlocale,
+        'getpreferredencoding',
+        return_value=encoding_to_use,
+    )
+    history_patch = patch.object(
+        rotki.events_historian,
+        'get_history',
+        lambda start_ts, end_ts, has_premium: ('', [EvmEvent(
+            event_identifier=GENESIS_HASH,
+            sequence_index=0,
+            timestamp=TimestampMS(1569924574000),
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(1)),
+            notes='Κοκκινολαίμης',  # Return an event with unicode characters
```

Comment:
Sorry, I just recalled that lefteris mentioned that greek letters have a lower range in the unicode set and this could be a reason for the export to work with greek letters and not Chinese ones. Could you try adding also one of those to this note? https://www.khngai.com/chinese/charmap/tbluni.php?page=0